### ISSUE TITLE
PyPI does not work with user/pass any more

### ISSUE THREAD

**Issue by AndreyNikiforov** [icloud-photos-downloader/icloud_photos_downloader#763]
# Problem

Latest release [failed to publish to pypi](https://github.com/icloud-photos-downloader/icloud_photos_downloader/actions/runs/7397772385/job/20127432450) with error:
```
HTTPError: 403 Forbidden from https://upload.pypi.org/legacy/          
         Username/Password authentication is no longer supported. Migrate to API
         Tokens or Trusted Publishers instead. See                              
         https://pypi.org/help/#apitoken and                                    
         https://pypi.org/help/#trusted-publishers
```

# Solution

Need to create token. I don't have current pypi user/pass -- they are stored as Github secrets and cannot be read. @ndbroadbent or @menkej (whoever has user/pass for pypi) can you please:
- create an org in pypi for icloudpd project, similar to github, npm
- add my 'andreynikiforov' account as admin to that org

Once that is done, I'll create token and switch icloudpd release to it. TIA

**Comment by menkej** [comment]
I take care.

**Comment by menkej** [comment]
I changed pypi to use token auth. I had to enable 2FA on the way. I have recovery codes and stuff stored in my personal keepass. Maybe we should find a more project specific place to safe this stuff... npm is still to go...

```
npm ERR! code 128
npm ERR! An unknown git error occurred
npm ERR! command git --no-replace-objects ls-remote ssh://git@github.com/dist/icloudpd.git
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
```

**Comment by AndreyNikiforov** [comment]
> I changed pypi to use token auth. I had to enable 2FA on the way. I have recovery codes and stuff stored in my personal keepass. Maybe we should find a more project specific place to safe this stuff... npm is still to go...
> 
> ```
> npm ERR! code 128
> npm ERR! An unknown git error occurred
> npm ERR! command git --no-replace-objects ls-remote ssh://git@github.com/dist/icloudpd.git
> npm ERR! git@github.com: Permission denied (publickey).
> npm ERR! fatal: Could not read from remote repository.
> npm ERR! 
> npm ERR! Please make sure you have the correct access rights
> npm ERR! and the repository exists.
> ```

Thanks for mitigating. How did you use new token? I still see user/pass in secrets and in code...

NPM is a separate issue and I am working on it.

**Comment by menkej** [comment]
> Thanks for mitigating. How did you use new token? I still see user/pass in secrets and in code...
I just followed the docs. pypi states use "__username__" als USER and the token as password. So I just updated the secrets accordingly. Just to save me to change the pipeline...

**Comment by menkej** [comment]
And then I just rerun the pipeline step and it worked. The current version should be on pypi now.
